_posts/2021-04-15-testing-react-tracking-with-jest-and-enzyme.markdown

```diff
@@ -0,0 +1,242 @@
+---
+layout: epic
+title: "Testing React Tracking with Jest and Enzyme"
+date: 2021-04-15
+categories: [testing, tracking, volt, cms, enzyme, react-tracking]
+author: matt-dole
+comment_id: 680
+---
+
+Recently, I needed to test a button that would make a tracking call using
+[react-tracking](https://github.com/NYTimes/react-tracking) and then navigate to a new page in a callback. This
+presented some challenges - I wasn't sure how to create a mocked version of react-tracking that would allow a
+callback to be passed.
+
+With some help from fellow Artsy engineers Chris Pappas and [Pavlos Vinieratos](https://twitter.com/pvinis), I got
```

Comment:
```suggestion
With some help from fellow Artsy engineers [Christopher Pappas](https://github.com/damassi) and [Pavlos Vinieratos](https://twitter.com/pvinis), I got
```

Reply:
thanks for this! I wasn't sure how best to tag you :)